### DISCUSSION TITLE
Installing Snipe-IT using php 8.0

### DISCUSSION THREAD

**Discussion by bdbj23** [snipe/snipe-it#12484] [General]
Is it possible to install snipe-it application using php 8.0? Everything I see says php 7.4 but I want to use 8.0 and am getting a 500 internal server error on my Windows 2019 server VM. Is there something I am missing or need to added in order to get the installation working corrrectly?

**Comment by inietov** [comment]
Hello there! You should be able to use it, yes. Did you happen to know what actual error it throws at you?

**Reply by bdbj23** [reply]
It seems like I am getting a few different 500 errors when attempting to access via we browser. In the logs I see 500 0 0 132028 and 500 0 0 9 and 500 0 0 8.
